docker/Dockerfile_centos

```diff
@@ -17,28 +17,23 @@ LABEL maintainer="AliceVision Team alicevision-team@googlegroups.com"
 ENV AV_DEV=/opt/AliceVision_git \
     AV_BUILD=/tmp/AliceVision_build \
     AV_INSTALL=/opt/AliceVision_install \
-    AV_BUNDLE=/opt/AliceVision_bundle \
-    PATH="${PATH}:${AV_BUNDLE}" \
     VERBOSE=1
 
-COPY . "${AV_DEV}"
+COPY CMakeLists.txt *.md ${AV_DEV}/
+COPY src ${AV_DEV}/src
 
 WORKDIR "${AV_BUILD}"
 
-RUN cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS:BOOL=ON -DTARGET_ARCHITECTURE=core \
+RUN cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS:BOOL=ON -DTARGET_ARCHITECTURE=${TARGET_ARCHITECTURE} \
       -DALICEVISION_BUILD_DEPENDENCIES:BOOL=OFF \
       -DCMAKE_PREFIX_PATH:PATH="${AV_INSTALL}" \
-      -DCMAKE_INSTALL_PREFIX:PATH="${AV_INSTALL}" -DALICEVISION_BUNDLE_PREFIX="${AV_BUNDLE}" \
+      -DCMAKE_INSTALL_PREFIX:PATH="${AV_INSTALL}" \
       -DALICEVISION_USE_ALEMBIC=ON -DMINIGLOG=ON -DALICEVISION_USE_CCTAG=ON -DALICEVISION_USE_OPENCV=ON -DALICEVISION_USE_OPENGV=ON \
       -DALICEVISION_USE_POPSIFT=ON -DALICEVISION_USE_CUDA=ON -DALICEVISION_BUILD_DOC=OFF -DALICEVISION_BUILD_EXAMPLES=OFF \
       "${AV_DEV}" && \
-\
-make install && \
-\
-make bundle && \
-\
-cd /opt && \
-\
-rm -rf "${AV_BUILD}"
+    make -j"$(nproc)" && \
+    make install && \
+    rm -rf "${AV_DEV}" "${AV_BUILD}" && \
+    echo "export ALICEVISION_SENSOR_DB=${AV_BUNDLE}/share/aliceVision/cameraSensors.db" >> /etc/profile.d/alicevision.sh
```

Comment:
and AV_BUNDLE is still used here

Reply:
That's a mistake, it should point to AV_INSTALL